client_folder_react_app/ethisim/src/pages/signup.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  TextField,
+  Box,
+  Typography,
+  Container,
+} from '@material-ui/core';
+import Copyright from '../components/copyright';
+
+export default function Signup() {
+  const useStyles = makeStyles((theme) => ({
```

Comment:
*Nitpick* Generally useStyles goes above / outside of the functional component. Another way some people prefer is to make a directory for each component, which includes the component file, the makeStyles file, and the test file